ollama run does not support tee

```
~$ ollama run phi | tee llms/out.txt
>>> Error getting size: inappropriate ioctl for device
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x9ad54e]

goroutine 1 [running]:
github.com/jmorganca/ollama/readline.(*Buffer).IsEmpty(...)
        /go/src/github.com/jmorganca/ollama/readline/buffer.go:309
github.com/jmorganca/ollama/readline.(*Instance).Readline(0xc00058a160)
        /go/src/github.com/jmorganca/ollama/readline/readline.go:100 +0x20e
github.com/jmorganca/ollama/cmd.generateInteractive(0xc00045b200, {{0x7ffcd3b32552, 0x3}, {0x0, 0x0}, {0x0, 0x0}, {0x1184c8c0, 0x0, 0x0}, ...})
        /go/src/github.com/jmorganca/ollama/cmd/interactive.go:185 +0x1cf
github.com/jmorganca/ollama/cmd.RunHandler(0xc00045b200, {0xc00044d020, 0x1, 0x1})
        /go/src/github.com/jmorganca/ollama/cmd/cmd.go:212 +0x6c5
github.com/spf13/cobra.(*Command).execute(0xc00045b200, {0xc00044cff0, 0x1, 0x1})
        /root/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x87c
github.com/spf13/cobra.(*Command).ExecuteC(0xc00045a900)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:985
main.main()
        /go/src/github.com/jmorganca/ollama/main.go:11 +0x4d
 ```
I tried using tee to save the output but this caused a segfault. 
Seems like the issue might be here?
https://github.com/ollama/ollama/blob/0ded7fdc4b33801fd0115656927b6097f800c544/readline/readline.go#L89
NewBuffer can be returning nil and there is no check for this. 
https://github.com/ollama/ollama/blob/0ded7fdc4b33801fd0115656927b6097f800c544/readline/buffer.go#L23
this is where NewBuffer is returning the nil
I do not understand why term.Getsize would be failing?

Note: it works perfectly when I do not use tee

--- a/readline/buffer.go
+++ b/readline/buffer.go
@@ -19,10 +19,9 @@ type Buffer struct {
 
 func NewBuffer(prompt *Prompt) (*Buffer, error) {
 	fd := int(os.Stdout.Fd())
-	width, height, err := term.GetSize(fd)
-	if err != nil {
-		fmt.Println("Error getting size:", err)
-		return nil, err
+	width, height := 80, 24
+	if termWidth, termHeight, err := term.GetSize(fd); err == nil {
+		width, height = termWidth, termHeight
 	}
 
 	lwidth := width - len(prompt.prompt())

